percent_encoding/src/lib.rs

```diff
@@ -77,6 +78,11 @@ const ASCII_RANGE_LEN: usize = 0x80;
 const BITS_PER_CHUNK: usize = 8 * mem::size_of::<Chunk>();
 
 impl AsciiSet {
+    /// An empty set.
+    pub const EMPTY: AsciiSet = AsciiSet {
```

Comment:
This seems like it's now inconsistent with the existing consts being and fucntion taking `&'static AsciiSet`.

Reply:
Yeah, I wasn't 100% sure about that. I went with EMPTY being a constant on the AsciiSet as empty seems like an inherent property of a type, but the other constants seem like usages of AsciiSet. The rationale for making the constants references rather than just values all seemed odd to me. What was that necessary for?